html-forms/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> HTML Forms</title>
+  </head>
+
+  <body>
+    <form>
+      <p> Email
+          <br />
+            <input type="text" name="Email" size="15"
+             max length="30"/>
+             <br />
+          Why is coding cool?
+              <br />
+            <select name="Why is coding cool">
+              <option value="Select an Option">Select a Value</option>
+              <option value="Because it's interesting">Because it's Interesting</option>
+              <option value="Because it's fun">Because it's fun</option>
+              <option value="Because it's powerful">Because it's powerful</option>
+            </select>
+            <br />
+          Favorite Language (choose one) <br />
+          <input type="radio" name="favorite language" value="HTML" /> HTML
```

Comment:
The text for these should be in a `label` element

Reply:
![image](https://user-images.githubusercontent.com/69921839/105094311-f2ad5600-5a58-11eb-981d-d2de61cb72cc.png)